src/app/app.module.ts

```diff
@@ -1,16 +1,69 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {NgModule} from '@angular/core';
 
-import { AppComponent } from './app.component';
+import {AppComponent} from './app.component';
+import {HeaderComponent} from './public-view/header/header.component';
+import {HomeModule} from './public-view/home/home.module';
+import {HTTP_INTERCEPTORS, HttpClientModule} from '@angular/common/http';
+import {FooterComponent} from './public-view/footer/footer.component';
+import {IvyGalleryModule} from 'angular-gallery';
+import {RouterModule, Routes} from '@angular/router';
+import {HomeComponent} from './public-view/home/home.component';
+import {NotFoundComponent} from './not-found/not-found.component';
+import {AuthGuardService} from './auth/services/auth-guard.service';
+import {PublicViewComponent} from './public-view/public-view.component';
+import {AdminViewModule} from './admin-view/admin-view.module';
+import {LoginComponent} from './admin-view/login/login.component';
+import {AuthInterceptorService} from './auth/services/auth-interceptor.service';
+import {AdminViewComponent} from './admin-view/admin-view.component';
+
+const routes: Routes = [
+  {
+    path: '', component: PublicViewComponent, children: [
+      {path: '', component: HomeComponent},
+      //TODO
+      // {path: 'projekty', component: ProjectsComponent},
```

Comment:
toto ma byt v ramci tohto PR? alebo v ramci ineho tasku sa dokonci?

Reply:
To sa bude riešiť vrámci tasku na zobrazenie projektov nechal som to tam kvôli tomu že som robil už Router